srcpkgs/highlight/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'highlight'
 pkgname=highlight
-version=3.56
+version=3.57
 revision=1
 build_style=gnu-makefile
 make_cmd="make QMAKE=qmake"
```

Comment:
looking into the Makefile, this isn't used...

Reply:
Removed :)